### PR TITLE
feat: Add dev instructions for handling failing PR tests

### PR DESCRIPTION
**Pull request by sarina**
RST translation of https://docs.google.com/document/d/129cyHROlSXJRpt8O8TuqQTr6uVi-C7aY_6NgopFnq5k/edit#, which in turn was created from discussion around the new PR manager role. This document is intended to help give pull request authors a few tools to begin diagnose test failures on their own. @feanil

### CODE REVIEW

**Comment by sarina** [comment]
@itsjeyd @mphilbrick211 this is a set of instructions for handling failing tests. Could you review for clarity (as I intend you to link these documents when helping people, it makes sense for you to understand them)?

@Carlos-Muniz do you think you could review for accuracy and clarity?

**Comment by mphilbrick211** [comment]
> @itsjeyd @mphilbrick211 this is a set of instructions for handling failing tests. Could you review for clarity (as I intend you to link these documents when helping people, it makes sense for you to understand them)?
> 
> @Carlos-Muniz do you think you could review for accuracy and clarity?

Looks good to me!

File: source/developers/references/running_pr_tests.rst [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+The first time you make a pull request to a repository you haven't made a PR to before, the tests will not run automatically. Someone who is a maintainer of or committer to the repo needs to validate your code is not dangerous before running the tests. If no one has looked at your PR over a few business days, the Community Contribution Management team can help move things along; tag them in a comment in your PR with a message and the handle: ``@openedx/community-pr-managers``
+
+The openedx/cla Check is Failing
+********************************
+
+If the openedx/cla check is failing:
+
+.. thumbnail:: /_images/failing_cla_check.png
+
+this means that we don't have a CLA (Contributor License Agreement) on file for you. To sign the CLA, please visit `openedx.org/cla`_. Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the CLA check by editing the PR title, pushing a new commit, or rebasing.  If the problem persists, you can tag the ``@openedx/cla-problems`` team in a comment on your PR for further assistance.
```
Comment (Carlos-Muniz) [inline source/developers/references/running_pr_tests.rst:22]:
```suggestion
This means that we don't have a CLA (Contributor License Agreement) on file for you. To sign the CLA, please visit `openedx.org/cla`_. Once you've signed the CLA, please allow 1 business day for it to be processed.  After this time, you can re-run the CLA check by editing the PR title, pushing a new commit, or rebasing.  If the problem persists, you can tag the ``@openedx/cla-problems`` team in a comment on your PR for further assistance.
```

**Review by Carlos-Muniz — COMMENTED** [review]
Content looks good, but it needs to be formatted. @mphilbrick211 suggested some changes in the google doc that should be integrated into the `running_pr_tests.rst` as well.

File: source/developers/references/running_pr_tests.rst [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+When you submit a pull request (PR) to an Open edX repository, a series of tests - which vary from repo to repo - are run. Sometimes tests won't be run at all, and sometimes they'll fail or be canceled. That looks like this:
+
+.. thumbnail:: /_images/failing_pr_tests.png
+
+This guide will help you start diagnosing errors with tests, but it is difficult to make it comprehensive. Post in the `Open edX Forums`_ under the "Development" category if you are running into issues you can't solve.
+
+Tests aren't running on my PR
+*****************************
+
+The first time you make a pull request to a repository you haven't made a PR to before, the tests will not run automatically. Someone who is a maintainer of or committer to the repo needs to validate your code is not dangerous before running the tests. If no one has looked at your PR over a few business days, the Community Contribution Management team can help move things along; tag them in a comment in your PR with a message and the handle: ``@openedx/community-pr-managers``
```
Comment (itsjeyd) [inline source/developers/references/running_pr_tests.rst:13]:
```suggestion
The first time you make a pull request to a repository you haven't made a PR to before, the tests will not run automatically. Someone who is a maintainer of or committer to the repo needs to validate your code is not dangerous before running the tests. If no one has looked at your PR over a few business days, the Community Contribution Management team can help move things along; tag them in a comment in your PR with a message and the handle: ``@openedx/community-pr-managers``.
```

**Review by itsjeyd — APPROVED** [review]
@sarina This looks good to me too 👍 Thanks for the ping!

**Comment by sarina** [comment]
@itsjeyd @mphilbrick211 you can now find this document here: https://docs.openedx.org/en/latest/developers/references/running_pr_tests.html and you can link it to newcomers who might be having problems with their tests.

**Comment by itsjeyd** [comment]
@sarina Awesome! 🙂

**Pull request merged**